### ISSUE TITLE
Hydrometry

### ISSUE THREAD

**Issue by goldma99** [goldma99/beavers#17]
[SEPA](https://www.sepa.org.uk/) maintains a network of [hydrometric monitoring stations](https://timeseriesdoc.sepa.org.uk/) across Scotland, since the early 20th Century. 

The goals of this branch are to
- [x] Determine the breadth and spatial density of monitoring stations in the Tayside region
- [x] Determine what measurement time series are available during our period of interest (~1990 -> 2020)
   - Parameters like water quality, water level, and other flood related items would be most relevant 
- [x] Determine quality of the data (i.e., do we have a close to balanced panel across all major river segments?
   - If temporal coverage is sub-par, consider averaging groups of years to get an aggregate sense of water levels before and after beaver introduction.

**Comment by goldma99** [comment]
49613a6

**Comment by goldma99** [comment]
17d55d0

* Level timeseries are available for 90 monitoring stations within the survey area bounding box parishes.
* Panel balance looks good. The difference between monthly records was always one month (i.e., I couldn't find any gaps)
* A potential downside is that the start dates of the timeseries vary quite a bit. A few start before 1970, while about 20 start after 2000. The latest "from" date in 2018. I have retained the timeseries with even the most recent from dates, because they could still provide pre-beaver information, depending on where they are located (i.e., if the timeseries began in 2018, but beavers didn't move into their river segment until after that, then they still provide pre- and post-treatment data).
* I included quality codes in the data request, but have not inspected them yet. I likely will use all the data values, and then run robustness checks with the low quality observations removed. There are a lot of observations that are marked as "unchecked - imported from legacy database," which I think is going to be true for most of the pre-beaver observations, since their current database probably wasn't established until after the internet.
